node/resolvers/providerLogo.ts

```diff
@@ -0,0 +1,16 @@
+import { Image } from 'vtex.geolocation-graphql-interface'
+
+// must be the same as assets/logo.png
+const SRC_URL =
+  'https://user-images.githubusercontent.com/26108090/100493516-45802680-3116-11eb-9028-06c336a587a7.png'
```

Comment:
we should host this on a VTEX server, can you upload this using the assets builder?

Reply:
lol, just commented about this. I will bring the discussion to [this comment](https://github.com/vtex-apps/google-geolocation-resolver/pull/4/files#r535254963), if you don't mind.